webapp/src/containers/modals/SubscribeModal/index.tsx

```diff
@@ -41,8 +40,9 @@ const SubscribeModal = () => {
         makeApiRequestWithCompletionStatus,
         state,
     } = usePluginApi();
-    const {visibility} = getSubscribeModalState(state);
-    const {currentTeamId} = useSelector((reduxState: GlobalState) => reduxState.entities.teams);
+    const {visibility, project, organization} = getSubscribeModalState(state);
+    const {currentTeamId} = useSelector((pluginState: GlobalState) => pluginState.entities.teams);
```

Comment:
```suggestion
    const {currentTeamId} = useSelector((state: GlobalState) => pluginState.entities.teams);
```
I don't think the `pluginState` is the right variable name here.

Reply:
And what about the previous variable `reduxState`?

Reply:
yes can use that